honeybee_doe2/cli/translate.py

```diff
@@ -19,20 +19,24 @@ def translate():
 @click.argument('hb-json', type=click.Path(
     exists=True, file_okay=True, dir_okay=False, resolve_path=True)
 )
+@click.option('--hvac_grouping', '-hg',
```

Comment:
What if we call it `havc_mapping` instead? My understanding is that this feature maps each room, story, or the whole model to HVAC systems.

Reply:
done! [see commit](https://github.com/ladybug-tools/honeybee-doe2/pull/59/commits/418982bc77511d6e557174a1d372de82fb722fdb)